awx_collection/plugins/doc_fragments/auth_plugin.py

```diff
@@ -68,6 +68,13 @@ class ModuleDocFragment(object):
         why: Collection name change
         alternatives: 'CONTROLLER_VERIFY_SSL'
     aliases: [ validate_certs ]
+  request_timeout:
+    description:
+    - Specify the timeout Ansible should use in requests to the controller host.
+    - Defaults to 10s, but this is handled by the shared module_utils code
```

Comment:
I don't think we need to specify that this is controlled by the module_utils code. All of these settings are.

Also.... we have a situation here (I think). This auth file was shared by both `ControllerAWXKitModule` and `ControllerAPIModule` but I don't think this new setting is respected by `ControllerAWXKitModule`. So we should consider that.... do we want to split the doc fragment into two, do we want to see if awxkit can take a timeout and respect it, something else?

Reply:
I don't see awxkit supporting this, maybe we can just add a line qualifying that this won't work for certain modules.

Reply:
Updated  description lines
```
    description:
    - Specify the timeout Ansible should use in requests to the controller host.
    - Defaults to 10 seconds
    - This will not work with the export or import modules.
```